tests/does-it-work/src/main.rs

```diff
@@ -23,16 +23,16 @@ fn run_test(rx: std::sync::mpsc::Receiver<()>) {
 #[cfg(test)]
 mod tests {
     use super::run_test;
-    use std::process::{Command, Stdio};
+    use std::process::Stdio;
     use std::sync::mpsc::channel;
     use std::thread;
+    use usdt_tests_common::root_command;
 
     #[test]
     fn test_does_it_work() {
         let (send, recv) = channel();
         let thr = thread::spawn(move || run_test(recv));
-        let dtrace = Command::new("sudo")
-            .arg("dtrace")
+        let dtrace = root_command("dtrace")
```

Comment:
I think as a result of sneaking my PR implementing JSON support in before this one, there's a similar catch in `tests/test-json/src/main.rs:99` ([here](https://github.com/oxidecomputer/usdt/blob/014f85ac97d0d923c17e5dcd51b9293e51acef7a/tests/test-json/src/main.rs#L99)). Would you mind changing that command invocation to `root_command` as well?

Reply:
I think I already got that one?

https://github.com/oxidecomputer/usdt/pull/25/files#diff-a1d523124420282a39e66d1a4de2a848cf274c4958c3efe0a1d657dbbfe2b687R100